clusterloader2/testing/load/job.yaml

```diff
@@ -9,6 +9,7 @@ metadata:
 spec:
   manualSelector: true
   parallelism: {{RandIntRange .ReplicasMin .ReplicasMax}}
+  completions: {{.Completions}}
```

Comment:
Personally I would simply write here:
```
# Set completions to value high enough, that we shouldn't never reach that
completions: 1000000
```

instead of doing this math and trying to calculate "right" value -- it's hard to say which one is right :)

Reply:
Alternatively write `{{MultiplyInt .ReplicasMax 100}}` here

Reply:
I was thing about it to write big enough number, however this could generate strange errors when huge clusters with jobs only workload would be tested.

Cannot use `{{MultiplyInt .ReplicasMax 100}}` as this needs to be constant over the test. - ReplicasMax is derivered from `randomScaleFactor`, which changes for different actions.

Reply:
Now I understand the reasoning behind this implementation. Thanks for explanation.